src/app/datagrid/selection/selection.ts

```diff
@@ -31,10 +31,27 @@ export class DatagridSelectionDemo {
     example = EXAMPLE;
     users: User[];
     selected: User[] = [];
+    toAdd: User[] = [];
+    toDelete: User;
+    toEdit: User;
 
     constructor(private inventory: Inventory) {
         inventory.size = 10;
         inventory.reset();
         this.users = inventory.all;
     }
+
+    onDelete() {
+        this.toDelete = this.selected[0];
```

Comment:
In your example, the Delete button in the action bar allows batch deletes. This doesn't account for it.